database/blogPosts.ts

```diff
@@ -107,3 +107,15 @@ export async function getBlogPostsBySessionToken(sessionToken: string) {
   `;
   return blogPosts;
 }
+
+export async function getSpecialBlogPosts(id: number) {
+  const blogPosts = await sql<BlogPost[]>`
+    SELECT
+      *
+    FROM
+      blog_posts
+    WHERE
+      id = ${id}
+  `;
+  return blogPosts;
```

Comment:
since we know this will only ever return a single blog post, let's return that

I'll take this over

Reply:
Done in 24971a9